_posts/2016-06-09-experimental-design.markdown

```diff
@@ -1,5 +1,115 @@
 ---
 layout: post
 title:  "Experimental design"
-categories: jekyll update
+img: experimental-design.png
 ---
+Experimental design addresses the problem of how to collect data points (experiments) to better control certain
```

Comment:
"Certain sources of variance" is quite ambiguous. 

I guess the aim of experimental design is to reduce the epistemic uncertainty we have about our system. But let's not use that word because not many people (I don't think) will know what it means...

Reply:
exactly, I didn't want to use epistemic uncertainty because of that.